src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -137,11 +149,11 @@ public override void Accept(OperationVisitor visitor)
 
     internal partial class BoundIfStatement : IIfStatement
     {
-        IExpression IIfStatement.Condition => this.Condition;
+        IOperation IIfStatement.Condition => this.Condition;
 
-        IStatement IIfStatement.IfTrue => this.Consequence;
+        IOperation IIfStatement.IfTrue => this.Consequence;
```

Comment:
With the push to make all properties nouns, should we rename IfTrue and IfFalse? (I actually like IfTrue and IfFalse as they are more obvious than Consequence and Alternative, but I also liked Thrown, Locked, and Upon so I'm not sure my taste carries any weight over the guideline we're now trying to follow,)